worlds/kh1/__init__.py

```diff
@@ -0,0 +1,300 @@
+        if self.options.hundred_acre_wood:
+            non_filler_item_categories.append("Torn Pages")
+        for name, data in item_table.items():
+            quantity = data.max_quantity
+            if data.category not in non_filler_item_categories:
+                continue
+            if name in starting_worlds:
+                continue
+            if data.category == "Puppies":
+                if self.options.puppies == "triplets" and "-" in name:
+                    item_pool += [self.create_item(name) for _ in range(quantity)]
+                if self.options.puppies == "individual" and "Puppy" in name:
+                    item_pool += [self.create_item(name) for _ in range(0, quantity)]
+                if self.options.puppies == "full" and name == "All Puppies":
+                    item_pool += [self.create_item(name) for _ in range(0, quantity)]
+            elif name == "Atlantica":
+                if self.options.atlantica:
+                    item_pool += [self.create_item(name) for _ in range(0, quantity)]
+            elif name == "Mermaid Kick":
+                if self.options.atlantica:
+                    if self.options.extra_shared_abilities:
+                        item_pool += [self.create_item(name) for _ in range(0, 2)]
+                    else:
+                        item_pool += [self.create_item(name) for _ in range(0, quantity)]
+            elif name == "Crystal Trident":
+                if self.options.atlantica:
+                    item_pool += [self.create_item(name) for _ in range(0, quantity)]
+            elif name == "High Jump":
+                if self.options.extra_shared_abilities:
+                    item_pool += [self.create_item(name) for _ in range(0, 3)]
+                else:
+                    item_pool += [self.create_item(name) for _ in range(0, quantity)]
+            elif name == "Progressive Glide":
+                if self.options.extra_shared_abilities:
+                    item_pool += [self.create_item(name) for _ in range(0, 4)]
+                else:
+                    item_pool += [self.create_item(name) for _ in range(0, quantity)]
+            elif name == "End of the World":
+                if self.options.end_of_the_world_unlock.current_key == "item":
+                    item_pool += [self.create_item(name) for _ in range(0, quantity)]
+            elif name == "EXP Zero":
+                if self.options.exp_zero_in_pool:
+                    item_pool += [self.create_item(name) for _ in range(0, quantity)]
+            elif name not in prefilled_items:
+                item_pool += [self.create_item(name) for _ in range(0, quantity)]
+        
+        for i in range(self.determine_reports_in_pool()):
+            item_pool += [self.create_item("Ansem's Report " + str(i+1))]
+        
+        while len(item_pool) > total_locations:
+            item_pool.pop(0)
```

Comment:
Whoa whoa whoa, aren't some of these important progression items? Or are you intentionally making sure to put the least relevant items into the itempool last? Either way, I think you should add a `warn` here (logging module)

```
removed_item = item_pool.pop(0)
warn(f"Item {removed_item} had to be removed from {self.player_name}'s world due to insufficient location count")
```